### DISCUSSION TITLE
activate frontend in zigbee2mqtt docker container

### DISCUSSION THREAD

**Discussion by backupandforkacc** [Koenkk/zigbee2mqtt#15390] [Devices]
From what I have read, I have to activate the frontend. Does that also apply for the docker container?
I started the container and docker ps shows:
`efc4bff397eb   koenkk/zigbee2mqtt            "docker-entrypoint.s…"   2 weeks ago    Up 3 days               0.0.0.0:8087->8080/tcp, :::8087->8080/tcp                                                        zigbee2mqtt`

so container is running and uses external port 8087
When I try to access it via 192.168.103.44:8087 I get the error ("Fehler: Verbindung fehlgeschlagen")

The docu says I have to activate the frontend, but it seems there is no bash in the container.
```
$ docker exec -it zigbee2mqtt bash
OCI runtime exec failed: exec failed: unable to start container process: exec: "bash": executable file not found in $PATH: unknown
```

and I also fail to get the configuration file from the container.
```
$ docker cp zigbee2mqtt:/opt/zigbee2mqtt/data/configuration.yaml /home/simon/Downloads
Error: No such container:path: zigbee2mqtt:/opt/zigbee2mqtt/data/configuration.yaml
```

**Comment by backupandforkacc** [comment]
I found it: The configuration can be found in the mounted devices at /data/configuration.yaml

**Comment by manoloxxl** [comment]
But how to fix the 

`OCI runtime exec failed: exec failed: unable to start container process: exec: "bash": executable file not found in $PATH: unknown`

No way to get access by console?
